modules/ROOT/pages/database-administration/composite-databases/alter-composite-databases.adoc

```diff
@@ -0,0 +1,26 @@
+:description: This page describes how to alter composite databases.
+[role=enterprise-edition not-on-aura]
+[[composite-databases-alter]]
+= Alter composite databases
+
+Composite databases can be altered using `ALTER DATABASE`.
```

Comment:
do we want to have a link to the alter standard database page here?

we don't seem to have links to the create/delete standard db pages from the composite db pages so maybe not 🤷